### PR TITLE
Allow words at the beginning of descriptions such as 'Always'

### PR DESCRIPTION
**Pull request by bl-ue**
See https://github.com/tldr-pages/tldr/pull/5438

### CODE REVIEW

**Review by sbrl — APPROVED** [review]
1 character change lol, but it does do the job :P

**Comment by waldyrious** [comment]
Can anyone explain the logic behind the regex in more detail? I've looked at a9b22cc6905c27d9abba8b363300c0ff6c869de1 and 9f9fe8905e69783cf4aba26c14489b1a21e426a5 but it's not entirely clear what the expression is trying to match in human-readable terms.

**Comment by bl-ue** [comment]
@waldyrious:
1. Any word at the start of a sentence that ends with "ing" and is 6 characters long or more is likely a verb in the gerund (e.g. executing, writing)
2. Any word at the start of a sentece that doesn't end with "us", "ss", or "ys" (e.g. superfluo**us**, succe**ss**, scenari**os**)

Make sense?

**Comment by waldyrious** [comment]
With examples, the intent is much clearer, thanks. Can you add a code comment explaining exactly what you wrote above?

**Comment by bl-ue** [comment]
Done, @waldyrious.

File: tldr.l [hunk 1]
```diff
@@ -151,6 +151,9 @@ space [ \t]
     yytext = this.matches[1];
     if (!yytext.match(/^[A-Z[]/)) yy.error(yylloc, 'TLDR015');
     if (this.matches[2] !== ':') yy.error(yylloc, 'TLDR005');
+    // Try to ensure that verbs at the beginning of the description are in the infinitive tense
+    // 1. Any word at the start of a sentence that ends with "ing" and is 6 or more characters long is likely a verb in the gerund (e.g. executing, writing)
+    // 2. Any word at the start of a sentence that doesn't end with "us", "ss", or "ys" is likely a verb in the present tense (e.g. superfluous, success, always)
```
Comment (waldyrious) [inline tldr.l:156]: The parenthesis are misplaced :)

```suggestion
    // 2. Any word at the start of a sentence that doesn't end with "us", "ss", or "ys" (e.g. superfluous, success, always) is likely a verb in the present tense
```

**Review by waldyrious — APPROVED** [review]
Looks great, thanks @bl-ue!

**Comment by MasterOdin** [comment]
It feels like there should be test cases here to cover the change in regex.

**Comment by sbrl** [comment]
For completeness, here's a railroad diagram for the new regex:

image

(uploaded PNG since GitHub doesn't support SVGs in comments :-/)

(diagram generated using [Regexper](https://regexper.com/#%2F%28%5E%5BA-Za-z%5D%7B3%2C%7Ding%20%29%7C%28%5E%5BA-Za-z%5D%2B%5B%5Eusy%5Ds%20%29%2F))

**Comment by bl-ue** [comment]
Oh yes, I kept forgetting to do this.

**Comment by MasterOdin** [comment]
@bl-ue yeah that would work.

**Review by owenvoke — APPROVED** [review]

**Review by MasterOdin — APPROVED** [review]

**Pull request merged**